src/components/utils/admonition.svelte

```diff
@@ -4,16 +4,16 @@
   export let expiration: number;
 
   const isExpired = new Date().valueOf() > expiration;
-  const icon = `../assets/badges/${path}.svg`;
+  const icon = `../assets/badges/with-shadow/${path}.svg`;
```

Comment:
Here, I'd keep the `with-shadow` out of this path. The component would be used something like `<Admonition picture="with-shadow/nodebuds.svg" />`.

Reply:
Isn't that already what the path line is for?  We dont need to specify like that when we can do it automatically by presetting `with-shadow` in the path

Reply:
If we want the behavior to always use path badges, then the method you've proposed works great. However, we may want to use badges that do not use shadows. Or to use any other static image on the site.

The plan is to use this feature to automate announcements in which case we'd want to be able to customize which icon is used for the announcement.